scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -288,10 +329,9 @@ def validate_motr_rpm(self):
     kernel_ver = op.replace('\n', '')
     check_type(kernel_ver, str, "kernel version")
 
-    if not self.k8:
-        kernel_module = f"/lib/modules/{kernel_ver}/kernel/fs/motr/m0tr.ko"
-        self.logger.info(f"Checking for {kernel_module}\n")
-        validate_file(kernel_module)
+    kernel_module = f"/lib/modules/{kernel_ver}/kernel/fs/motr/m0tr.ko"
```

Comment:
kernel modules are required for lnet, which is not supported in k8 currently, in k8 we support only libfab.  Removing of lnet or decision to support it will be taken care in another tkt.